Products/CMFPlone/tests/test_okay.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from plone.testing.z2 import Browser
+from Products.CMFPlone.testing import PRODUCTS_CMFPLONE_INTEGRATION_TESTING
+
+import unittest
+
+
+class OkayTest(unittest.TestCase):
+    """Test the OK simple status view.
```

Comment:
I sometimes see a blank line above it, and I really do not like it.
But I will make it one line indeed.